### PR TITLE
[12.0] postlogistics sanitize string

### PR DESCRIPTION
**Pull request by yvaucher**
sanitize strings

Some characters are not handled by the REST API.

### CODE REVIEW

**Review by i-vyshnevska — APPROVED** [review]

File: delivery_carrier_label_postlogistics/postlogistics/web_service.py [hunk 1]
```diff
@@ -17,6 +17,12 @@
 _compile_itemnum = re.compile(r'[^0-9]')
 
 
+def _sanitize_string(string):
+    if not string:
+        return False
```
Comment (guewen) [inline delivery_carrier_label_postlogistics/postlogistics/web_service.py]: is the original string a boolean in this case? I think it makes more sense to return `""`

File: delivery_carrier_label_postlogistics/postlogistics/web_service.py [hunk 2]
```diff
@@ -17,6 +17,12 @@
...
+    return string.replace("\u2018", "'").replace("\u2019", "'")
```
Comment (guewen) [inline delivery_carrier_label_postlogistics/postlogistics/web_service.py]: Do they have a list of characters they support? I guess that's not the only ones they don't support?

**Comment by yvaucher** [comment]
closing in favor of PR #309

**Comment by yvaucher** [comment]
reopening it as #309 doesn't seems to work

File: delivery_carrier_label_postlogistics/postlogistics/web_service.py (hunk 2 above)
Comment (yvaucher) [reply delivery_carrier_label_postlogistics/postlogistics/web_service.py]: I asked for a list but I got only a vague answer about using utf-8 encoding.

**Pull request merged**